src/main/webapp/consentGroupDocuments/ConsentGroupDocuments.js

```diff
@@ -6,9 +6,9 @@ import { ConfirmationDialog } from "../components/ConfirmationDialog";
 import { h, hh } from 'react-hyperscript-helpers';
 import '../index.css';
 import { AlertMessage } from "../components/AlertMessage";
-import { Spinner } from '../components/Spinner';
-
-export const ConsentGroupDocuments = hh(class ConsentGroupDocuments extends Component {
+import LoadingWrapper from "../components/LoadingWrapper";
```

Comment:
Run these changes through the eslint rules for the project - you should see double quotes come up as warnings and should be single quotes. For example, when I optimize imports for this file, this is what it does for me:
```
import { Component, Fragment } from 'react'
import { h, hh } from 'react-hyperscript-helpers'
import { AlertMessage } from '../components/AlertMessage'
import { ConfirmationDialog } from '../components/ConfirmationDialog'
import { Documents } from '../components/Documents'
import LoadingWrapper from '../components/LoadingWrapper'
import '../index.css'
import { ConsentGroup, DocumentHandler, User } from '../util/ajax'
import { CONSENT_DOCUMENTS } from '../util/DocumentType'
```

Reply:
I will incorporate this practice into my code. Right now i've optimize all imports of the files that i've worked on this PR.